docs/index.md

```diff
@@ -1,4 +1,4 @@
-# MyST - Markedly Structured Text
+# MyST - Markedly Structured Text <img src="_static/logo-square.png" width=40 />
```

Comment:
I see this one a little bit blurry, maybe?

![Screen Shot 2021-05-31 at 17 14 25](https://user-images.githubusercontent.com/1640669/120238983-b6260e00-c233-11eb-8fda-dbd02b6673f9.png)